book/payments.rst

```diff
@@ -77,7 +77,38 @@ You can have different payment methods using the same gateway, like PayPal or St
 Gateway and Configurations
 --------------------------
 
-...
+Sylius by default is configured to use offline payment method if you want to for example use paypal checkout express you need to configure it by your self.
```

Comment:
..use offline payment method, **but** if you want to **use some payment providers (e.g. paypal or stripe) you need to configure the appropriate gateways.** ?